### PR TITLE
book-store: Add styles for book store

### PR DESCRIPTION
**Pull request by yodit93**
### In this project, I
- [ ] added new JSX elements,
- [ ] used styles from Zeplin design,
- [ ] added styles to the bookstore to stick with Zeplin,

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+        <Link to="/" className="nav-link books">BOOKS</Link>
       </li>
       <li className="nav-item">
-        <Link to="/categories" className="nav-link">CATEGORIES</Link>
+        <Link to="/categories" className="nav-link categories">CATEGORIES</Link>
       </li>
     </ul>
+    <div className="Oval">
+      <div className="Mask">
+        <CgProfile />
```
Comment (rustam-tolipov) [inline src/components/Navbar.js:17]: **You put great effort into making this project.** 

However, we have some places that need some improvements. Let me show you the places to improve :)

- [ ] Kindly style the profile image as the original design:


Your image            |  Original image
:-------------------------:|:-------------------------:
Screenshot 2023-01-03 at 18 29 21 |

File: src/components/Form.js [hunk 2]
```diff
@@ -27,25 +27,26 @@ const Form = () => {
...
-      <h2 className="form-title">Add New Book</h2>
+      <div className="Line" />
+      <h2 className="form-title">ADD NEW BOOK</h2>
       <div className="form-rapper">
         <input
           type="text"
           className="add-book"
           name="title"
-          placeholder="Book title"
+          placeholder="BOOK TITLE"
```
Comment (rustam-tolipov) [inline src/components/Form.js:37]: - [ ] Also your form is not justified to its original place. Kindly make its width 100% placed as the original :)

Your image            |  Original image
:-------------------------:|:-------------------------:
Screenshot 2023-01-03 at 18 29 21 |

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @yodit93,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Clean code_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 2 above)
Comment (yodit93) [reply src/components/Form.js:37]: Thank you for your constructive comments. I will fix it.

File: src/components/Navbar.js (hunk 1 above)
Comment (yodit93) [reply src/components/Navbar.js:17]: Hello @Rustamxon7 
Thank you, I will update it.

**Review by andersonlebon — APPROVED** [review]
## Hi @yodit93 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yodit93** [comment]
Thank you!!!

**Pull request merged**